lib/storage/providers/SQLiteStorage.js

```diff
@@ -88,10 +88,9 @@ const provider = {
              ON CONFLICT DO UPDATE
              SET valueJSON = JSON_PATCH(valueJSON, JSON(:value));
         `;
-        const queryArguments = _.map(pairs, (pair) => {
-            const value = JSON.stringify(pair[1]);
-            return [pair[0], value];
-        });
+
+        const queryArguments = _.map(pairs, pair => [pair[0], _.isUndefined(pair[1]) ? null : pair[1]]);
```

Comment:
Can we filter out any pair that has an `undefined` value? I believe this would still delete it from storage.